### DISCUSSION TITLE
Removal of capitals in yourls

### DISCUSSION THREAD

**Discussion by bowsjb** [YOURLS/YOURLS#3053] [Q&A] [answered]
Hi - just installed Yourls through softaculous.

I tried an alias of DRD1a and it did shorten but to just 1a.  How can I keep the capitals?

**Comment by dgw** [comment, marked as answer]
See `YOURLS_URL_CONVERT` setting and [related FAQ entry](https://yourls.org/#FAQ;base3662).

**Reply by bowsjb** [reply]
Thank you.  Problem solved.
